src/rust/ide/tests/language_server.rs

```diff
@@ -62,6 +64,9 @@ wasm_bindgen_test_configure!(run_in_browser);
 // #[wasm_bindgen_test::wasm_bindgen_test(async)]
 #[allow(dead_code)]
 async fn ls_text_protocol_test() {
+    // FIXME broken, as cannot know the project name
```

Comment:
this should be marked as `FIXME[MWU]` and also the description should be better - right now I completely do not understand it.